azext_edge/edge/command_map.py

```diff
@@ -39,6 +39,7 @@ def load_iotops_commands(self, _):
         cmd_group.command("delete", "delete")
         cmd_group.command("clone", "clone_instance", is_preview=True)
         cmd_group.command("get-versions", "get_versions", is_experimental=True)
+        cmd_group.command("migrate-assets", "migrate_assets")
```

Comment:
Do you know how many versions we will have this? can we add in experimental or preview to give an easier time of deprecation?

Reply:
This would be applicable for as long as root assets are supported. Right now, this branch of CLI the package itself is preview (not available in existing GA CLI branch), once this branch GA's the API backing this also GA's. I don't think experimental is appropriate here, but I can explicitly add preview.